src/passes/Strip.cpp

```diff
@@ -62,6 +62,12 @@ Pass* createStripDebugPass() {
   });
 }
 
+Pass* createStripDWARFPass() {
+  return new Strip([&](const UserSection& curr) {
+    return curr.name.find(".debug") == 0 || curr.name.find("reloc..debug") == 0;
```

Comment:
is there really no better way to find dwarf sections than by name?

Reply:
Hmm, how else would you do it? Do they have a magic number in their content or something like that?